apps/web/src/app/[locale]/challenge/[id]/solutions/_components/_actions.ts

```diff
@@ -18,3 +19,11 @@ export async function postSolution({ challengeId, description, title, userId }:
     },
   });
 }
+
+export async function pinOrUnpinSolution(id: number, isPinned: boolean) {
+  await prisma.sharedSolution.update({
```

Comment:
add server side validation that the homie is admin plox

Reply:
@bautistaaa we can let this go as long as you promise to make a ticket for handling this repo wide?

Reply:
nah ill fix tonight!